### PR TITLE
Feature/color outputword selector

### LINKED ISSUE #21: Improve hover color on word selector
Currently the word selector within the output editor shows a red (ugly) color on focused items. It could be improved changing the color and maybe the description on the item. 

image

### PR DESCRIPTION
**Pull request by marroq**
imagen

### CODE REVIEW

**Review by baquiax — APPROVED** [review]
LGTM!

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (baquiax) [inline .vscode/launch.json]: is this file needed?

Comment (marroq) [reply .vscode/launch.json]: it's only for debug in visual code, it is already eliminated
